README.md

```diff
@@ -1,65 +1,48 @@
-  <a href="https://hub.docker.com/r/verdaccio/verdaccio/"><img alt="npm Downloads" src="https://img.shields.io/docker/pulls/verdaccio/verdaccio.svg?maxAge=43200"></a>
-  <a href="https://www.browserstack.com/automate/public-build/am8rVDJ1V1BSbW90aE4xUnZsbnNUOFVuVS9ZRmJWL3NWcjkzRUxycHF3TT0tLUh5aTBiY2VzT1A1NDRWdHg2L0FQT3c9PQ==--0e9064ebcb01bfebb9976a543500b0396c0bfcae%"><img src='https://www.browserstack.com/automate/badge.svg?badge_key=am8rVDJ1V1BSbW90aE4xUnZsbnNUOFVuVS9ZRmJWL3NWcjkzRUxycHF3TT0tLUh5aTBiY2VzT1A1NDRWdHg2L0FQT3c9PQ==--0e9064ebcb01bfebb9976a543500b0396c0bfcae%'/></a>
-  <a href="https://discord.gg/AwXRqPD"
-  ><img alt="discord" src="https://img.shields.io/badge/Discord-%23questions%20%23general-blue.svg"/>
-  </a>
-</div>
-<div align="center">
-  <p align="center"><img src="https://github.com/verdaccio/verdaccio/blob/master/assets/gif/verdaccio_big_30.gif?raw=true"></p>
-
-</div>
-
-<h2 align="center">Install</h2>
+![verdaccio logo](https://github.com/verdaccio/verdaccio/raw/master/assets/bitmap/verdaccio%402x.png)
 
-Install with npm:
+![verdaccio gif](https://github.com/verdaccio/verdaccio/blob/master/assets/gif/verdaccio_big_30.gif?raw=true)
 
-```bash
-npm install --global verdaccio
-```
+# Version 4 
 
-Install with yarn:
+[Verdaccio](https://verdaccio.org/) is a simple, **zero-config-required local private npm registry**. 
+No need for an entire database just to get started! Verdaccio comes out of the box with 
+**its own tiny database**, and the ability to proxy other registries (eg. npmjs.org), 
+caching the downloaded modules along the way. 
+For those looking to extend their storage capabilities, Verdaccio 
+**supports various community-made plugins to hook into services such as Amazon's s3, 
+Google Cloud Storage** or create your own plugin.
 
-```bash
-yarn global add verdaccio
-```
 
-Install with pnpm:
+![verdaccio (latest)](https://img.shields.io/npm/v/verdaccio/latest.svg)
+![verdaccio (next)](https://img.shields.io/npm/v/verdaccio/next.svg)
+![verdaccio (next)](http://img.shields.io/npm/dy/verdaccio.svg)
+![docker pulls](https://img.shields.io/docker/pulls/verdaccio/verdaccio.svg?maxAge=43200)
+
+![circle ci status](https://circleci.com/gh/verdaccio/verdaccio.svg?style=shield&circle-token=:circle-token)
+![codecov](https://img.shields.io/codecov/c/github/verdaccio/verdaccio/4.x.svg)
+![discord](https://img.shields.io/discord/388674437219745793.svg)
+![node](https://img.shields.io/node/v/verdaccio/latest.svg)
+![MIT](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![Crowdin](https://d322cqt584bo4o.cloudfront.net/verdaccio/localized.svg)](https://crowdin.com/project/verdaccio)
 
-```bash
-pnpm i -g verdaccio
-```
 
+![Twitter followers](https://img.shields.io/twitter/follow/verdaccio_npm.svg?style=social&label=Follow)
+![Github](https://img.shields.io/github/stars/verdaccio/verdaccio.svg?style=social&label=Stars)
```

Comment:
Some budgets are not linking to what they said. Is it the desired behavior?

Reply:
![screen shot 2018-10-21 at 9 00 47 pm](https://user-images.githubusercontent.com/558752/47271034-6871a100-d574-11e8-8ff6-9e8012133bf5.png)
It's not possible to link with markdown. I don't want to use HTML again. Any suggestion?